clerk/organizations.go

```diff
@@ -228,6 +229,11 @@ func (s *OrganizationsService) addMembersSearchParamsToRequest(r *http.Request,
 			query.Add("user_id", userID)
 		}
 	}
+	if params.Roles != nil {
+		for _, role := range params.Roles {
+			query.Add("user_id", role)
```

Comment:
:question: Is `"user_id"` intentional here? Shouldn't the parameter have to do with role?

Reply:
This was a mistake. i have replace it with role.